README.md

```diff
@@ -119,31 +45,54 @@ text file `_config/dirs.txt`, one per line. E.g.
     user@eris1n2.partners.org:/data/pnl
 
 * NOTE: replace user with whomever has sudo access.
-    
+
+Modify `html-report-generator/generate_usage_report.py` to change defaults in the html report.
+
 # Running
 
+## Scripts
     logbigfiles <dir> # finds files > 300M in <dir>, writes csv output to _data/logbigfiles/
+
     logdirsizes <dir> # writes directory sizes in `<dir>` (default depth 3) to `_data/logdirsizes/`
+
     logdirsizesall # calls `logdirsizes` for each directory in _config file `_config/dirs.txt`
+
     mydf # prints file system usage for each directory in _config file `_config/dirs.txt`
+
```

Comment:
Is there a reason you included all the white spaces above?

Reply:
Yes, I thought it formatted better on Github and provided more distinction between the scripts. Can revert if wanted.

Reply:
No need to revert.